tests/test_helpers/helpers.ts

```diff
@@ -265,7 +266,13 @@ export async function mountComponent<Props extends { [key: string]: any }>(
   model.drawLayer = () => {};
   const env = makeTestEnv({ ...optionalArgs.env, model: model });
   const props = optionalArgs.props || ({} as Props);
-  const app = new App(component, { props, env, test: true, translateFn: _t });
+  const app = new App(component, {
+    props,
+    env,
+    test: true,
+    translateFn: _t,
+    warnIfNoStaticProps: optionalArgs.warnIfNoStaticProps,
```

Comment:
Should I enable this for every `MountComponent` ? That menas that every random `Parent` component that we decalre in the tests would need to be correctly declared...

Reply:
I think we should. I already lost time multiple times trying to figure out why a test wouldn't pass just because we had some random invalid props that where either false from the very beginning or were not properly adapted.

Reply:
Sure, I added them to every test component then :)